packages/components/src/DataList/components/DataListActionsMenu/DataListActionsMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { CSSProperties, PropsWithChildren, useState } from "react";
+import { AnimatePresence, Variants, motion } from "framer-motion";
+import { tokens } from "@jobber/design";
+import { useFocusTrap } from "@jobber/hooks/useFocusTrap";
+import { useRefocusOnActivator } from "@jobber/hooks/useRefocusOnActivator";
+import { useOnKeyDown } from "@jobber/hooks/useOnKeyDown";
+import styles from "./DataListActionsMenu.css";
+
+interface DataListActionsMenuProps {
+  readonly visible: boolean;
+  readonly position: {
+    readonly x: number;
+    readonly y: number;
+  };
+  readonly onRequestClose: () => void;
+}
+
+const variants: Variants = {
+  hidden: { opacity: 0, y: -10 },
+  visible: { opacity: 1, y: 0 },
+};
+
+export function DataListActionsMenu({
+  visible = false,
+  position,
+  onRequestClose,
+  children,
+}: PropsWithChildren<DataListActionsMenuProps>) {
+  const [ref, setRef] = useState<HTMLDivElement | null>();
+
+  useRefocusOnActivator(visible);
+  const focusTrapRef = useFocusTrap<HTMLDivElement>(visible);
+  useOnKeyDown(onRequestClose, "Escape");
+
+  return (
+    <AnimatePresence>
+      {visible && (
+        <>
+          <div className={styles.overlay} onClick={onRequestClose} />
+
+          <motion.div
+            ref={setRef}
+            variants={variants}
+            initial="hidden"
+            animate="visible"
+            exit="hidden"
+            transition={{ duration: tokens["timing-base"] / 1000 }}
```

Comment:
You can use the `TRANSITION_DURATION_IN_SECONDS` const

Reply:
I knew there was 1 more haha. good catch!